src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -570,12 +566,28 @@ def connected_cluster_exists(client, resource_group_name, cluster_name):
     return True
 
 
-def get_config_dp_endpoint(cmd, location):
+def get_default_config_dp_endpoint(cmd, location):
     cloud_based_domain = cmd.cli_ctx.cloud.endpoints.active_directory.split('.')[2]
     config_dp_endpoint = "https://{}.dp.kubernetesconfiguration.azure.{}".format(location, cloud_based_domain)
     return config_dp_endpoint
 
 
+def get_config_dp_endpoint(cmd, location, values_file):
+    release_train = None
+    metadata = utils.get_metadata(cmd.cli_ctx.cloud.endpoints.resource_manager)
+    # Read and validate the helm values file for Dogfood.
+    config_dp_endpoint = None
+    if cmd.cli_ctx.cloud.endpoints.resource_manager == consts.Dogfood_RMEndpoint:
+        config_dp_endpoint, release_train = validate_env_file_dogfood(values_file, True)
```

Comment:
i believe you have removed the second argument from this function definition. 'True' should not be required here